helm/adelphi/templates/gemini-job.yaml

```diff
@@ -9,14 +9,14 @@ spec:
   templates:
   - name: gemini-oracle-sut
     container:
-      image: golang
+      image: adelphidev/gemini:1.7.3
       workingDir: /workspace
       command:
         - sh
         - -c
       args:
         - mkdir /results/gemini;
-          /workspace/gemini
+          /go/gemini
           --oracle-cluster={{ .Values.source.clusterName }}-{{ .Values.source.dc }}-service
           --oracle-username={{ .Values.source.clusterName }}-superuser
           --oracle-password="`cat /source-secret/password`"
```

Comment:
Is there a clean way to move all of these args into the Dockerfile as well?  Might be nice to avoid having to spell all this out within the YAML if we can do so without creating a configuration nightmare.

Reply:
Here we're passing args directly to the gemini cli. We can't really hardcode them in the image.